web/assets/main.scss

```diff
@@ -68,6 +68,10 @@ $govuk-assets-path: "/lpa/frontend/assets/";
   color: govuk-colour("red") !important;
 }
 
+.app-\!-colour-text-lighter {
```

Comment:
You don't need !- prefix as it should only be used for override classes (this file has made it confusing to follow and set a bad example because it's wrongly used in other places too). Also you'll need to add the class to dark mode and tweak the colour as its currently difficult to ready against the dark background

Reply:
I just changed this to white.